### PR TITLE
CQ-4349354 - Enable Omega Tracking - For portal components

### PR DESCRIPTION
**Pull request by TalmizAhmed**
## Description
Changed the `trackingFeature` string in the cq_dialog's `.content.xml` to the appropriate adaptiveform version of the string
Components affected:
- Accordion
- Title
 
Added the attribute to the following `formsportal` components
- `draftsandsubmissions`
- `link`
- `portallister`
- `searchlister`

## Related Issue
https://jira.corp.adobe.com/browse/CQ-4349354

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

File: ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/title/v1/title/_cq_dialog/.content.xml [hunk 1]
```diff
@@ -18,9 +18,9 @@
     jcr:primaryType="nt:unstructured"
     jcr:title="Title"
     sling:resourceType="cq/gui/components/authoring/dialog"
-    extraClientlibs="[core.wcm.components.title.v2.editor]"
+    extraClientlibs="[core.forms.components.title.v1.editor]"
```
Comment (nit23uec) [inline ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/title/v1/title/_cq_dialog/.content.xml:21]: there is no such library `core.forms.components.title.v1.editor` in the codebase.
ideally, this should come from wcm text inheritance implicitly via resourcesupertype. can you pls remove this and see if https://github.com/adobe/aem-core-wcm-components/blob/a7e377c6810dd2125a902d3ef3df5c92ec2a1719/content/src/content/jcr_root/apps/core/wcm/components/title/v2/title/clientlibs/editor/js/title.js is getting loaded in the browser while opening edit of text or not.

**Review by nit23uec — CHANGES_REQUESTED** [review]
inline comments.

**Review by nit23uec — APPROVED** [review]

**Comment by nit23uec** [comment]
seems some issue with rebase.. it contains changes unrelated to this PR. pls fix.

**Comment by TalmizAhmed** [comment]
@nit23uec Should be rebased properly now. Have a look

**Review by nit23uec — APPROVED** [review]

**Pull request merged**